### PR TITLE
Added documentation for the project globs

### PR DESCRIPTION
**Pull request by flavius-dinu**
# Description of the change

## Checklist

Please make sure that the proposed change checks all the boxes below before requesting a review:

- [x] I have reviewed the [guidelines for contributing](https://github.com/spacelift-io/user-documentation/blob/main/CONTRIBUTING.md) to this repository.
- [x] The preview looks fine.
- [x] The tests pass.
- [x] The commit history is clean and meaningful.
- [x] The pull request is opened against the `main` branch.
- [x] The pull request is no longer marked as a draft.
- [x] You agree to license your contribution under the [MIT license](https://github.com/spacelift-io/user-documentation/blob/main/LICENSE) to Spacelift (not required for Spacelift employees).
- You have updated the navigation files correctly:
    - [x] No new pages have been added, or;
    - [ ] Only _nav.yaml_ has been updated because the changes only apply to SaaS, or;
    - [ ] Only _nav.self-hosted.yaml_ has been updated because the changes only apply to Self-Hosted, or;
    - [ ] Both _nav.yaml_ and _nav.self-hosted.yaml_ have been updated.

If the proposed change is ready to be merged, please request a review from `@spacelift-io/solutions-engineering`. Someone will review and merge the pull request.

_Spacelift employees should request reviews from the relevant engineers and are allowed to merge pull requests after they got at least one approval._

Thank you for your contribution! 🙇

### CODE REVIEW

File: docs/concepts/stack/stack-settings.md [hunk 1]
```diff
@@ -180,6 +180,25 @@ Project root points to the directory within the repo where the project should st
...
 
+### Project globs
+
+Project globs allow you to specify files and directories outside of the project root that the stack cares about. In the absence of push policies, any changes made to the project root and any paths specified by project globs will trigger Spacelift runs.
+
+![](../../assets/screenshots/Stack_Settings_Project_Globs.png)
+
+You aren't required to add any project globs if you don't want to, but you have the option to add as many project globs as you want for a stack.
+
+Under the hood, project globs take advantage of the [filepath.Match](https://pkg.go.dev/path/filepath#Match) function to do pattern matching.
```
Comment (tomasmik) [inline docs/concepts/stack/stack-settings.md:191]:
```suggestion
Under the hood, project globs takes advantage of the [filepath.Match](https://pkg.go.dev/path/filepath#Match) function to do pattern matching.
```

Comment (flavius-dinu) [reply docs/concepts/stack/stack-settings.md:191]: Under the hood, the project globs feature takes advantage

File: docs/concepts/stack/stack-settings.md [hunk 2]
```diff
@@ -180,6 +180,25 @@ Project root points to the directory within the repo where the project should st
 !!! info
     The project root can be overridden by the [runtime configuration](../configuration/runtime-configuration/README.md#project_root-setting) specified in the `.spacelift/config.yml` file.
 
+### Project globs
+
+Project globs allow you to specify files and directories outside of the project root that the stack cares about. In the absence of push policies, any changes made to the project root and any paths specified by project globs will trigger Spacelift runs.
```
Comment (eminalemdar) [inline docs/concepts/stack/stack-settings.md:185]:
```suggestion
Project globs option allows you to specify files and directories outside of the project root that the stack cares about. In the absence of push policies, any changes made to the project root and any paths specified by project globs will trigger Spacelift runs.
```

File: docs/concepts/stack/stack-settings.md (hunk 1 above)
Comment (eminalemdar) [inline docs/concepts/stack/stack-settings.md:191]:
```suggestion
Under the hood, project globs take advantage of the [filepath.Match](https://pkg.go.dev/path/filepath#Match){: rel="nofollow"} function to do pattern matching.
```

File: docs/concepts/stack/stack-settings.md [hunk 3]
```diff
@@ -182,13 +182,13 @@ Project root points to the directory within the repo where the project should st
...
 
-Project globs allow you to specify files and directories outside of the project root that the stack cares about. In the absence of push policies, any changes made to the project root and any paths specified by project globs will trigger Spacelift runs.
+The project globs option allows you to specify files and directories outside of the project root that the stack cares about. In the absence of push policies, any changes made to the project root and any paths specified by project globs will trigger Spacelift runs.
 
 ![](../../assets/screenshots/Stack_Settings_Project_Globs.png)
 
 You aren't required to add any project globs if you don't want to, but you have the option to add as many project globs as you want for a stack.
 
-Under the hood, project globs take advantage of the [filepath.Match](https://pkg.go.dev/path/filepath#Match) function to do pattern matching.
+Under the hood, the project globs feature takes advantage of the [filepath.Match](https://pkg.go.dev/path/filepath#Match){: rel="nofollow"} function to do pattern matching.
```
Comment (eminalemdar) [inline docs/concepts/stack/stack-settings.md:191]: feature or option? I think we should be consistent on these terms.

Comment (flavius-dinu) [reply docs/concepts/stack/stack-settings.md:191]: Let's stick with option.

**Review by eminalemdar — APPROVED** [review]
LGTM!

**Review by tomasmik — APPROVED** [review]

**Pull request merged**